addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -66,7 +66,7 @@
 
                         <field name="state" readonly="1" widget="statusbar"/>
                     </header>
-                    <div class="alert alert-info text-center" role="alert" attrs="{'invisible': ['&amp;','&amp;','&amp;',('sent', '=', 0),('ignored', '=', 0),('scheduled', '=', 0),('failed', '=', 0)]}">
+                    <div class="alert alert-info text-center" role="alert" attrs="{'invisible': ['&amp;','&amp;','&amp;', '|',('sent', '=', 0),('ignored', '=', 0),('scheduled', '=', 0),('failed', '=', 0),('state', '!=', 'in_queue')]}">
```

Comment:
Shouldn't the OR be at the beginning of the invisible domain ? Like invisible if state not in queue OR old condition ?

Reply:
Don't know if it change much. Work the same for me.